README.md

```diff
@@ -5,3 +5,16 @@ npm install mx-react-components
 
 ## Usage
 [See Documentation](http://moneydesktop.github.io/mx-react-components/)
+
+## Development
+
+To view your documentation changes before committing you can run the following commands:
+
+```
+npm install
+npm run build
```

Comment:
This command (which runs `webpack`) doesn't watch, do you think it would be a better workflow to have two console sessions running: one for webpack watch and one for jekyll server?